SAMPLE_CREDENTIAL_CONFIG.yml

```diff
@@ -1,48 +1,53 @@
 ---
 
-# Vault Password - Master Ansible Vault password used to encrypt all of the following variables
+# Vault Password - Master Ansible Vault password used to encrypt all of the following variables. 
+# Please store this password securely for future use as it does not persist after project execution.
 
 vault_password: <CHANGEME>
 
 # aws_credentials.yml
 
-aws_credential_list:
-  - "<CHANGEME>"
+aws_credential_list: <CHANGEME>
```

Comment:
It doesnt look like this supports a list? May be better to rename to `aws_credential_name` and explain that only one account can be specified in the documentation